src/udev/udev-builtin-blkid.c

```diff
@@ -321,8 +321,8 @@ static int builtin_blkid(struct udev_device *dev, int argc, char *argv[], bool t
         if (is_gpt)
                 find_gpt_root(dev, pr, test);
 
-        blkid_free_probe(pr);
 out:
+        blkid_free_probe(pr);
```

Comment:
Thanks for fixing the memory leak.

I think it's probably better to use
```c
_cleanup_blkid_free_probe_ blkid_probe pr = NULL;
```
instead of
```c
blkid_probe pr;
...
blkid_free_probe(pr);
```